docker/Dockerfile

```diff
@@ -352,3 +352,9 @@ RUN chmod 600 /home/nubots/.ssh/id_rsa /home/nubots/.ssh/config
 RUN install -d -m 0755 /home/nubots/NUbots
 RUN install -d -m 0777 /home/nubots/build
 WORKDIR /home/nubots/NUbots
+
+#Speech Rec
+RUN sudo chmod a+rw /opt
+RUN curl -L https://raw.githubusercontent.com/NUbots/NUbots/legge/speech-rec-module-and-role/doc/ArchInstall/arch-speechrec.sh -o /opt/arch-speechrec.sh
+RUN sudo chmod a+rw /opt/arch-speechrec.sh
+RUN /bin/bash /opt/arch-speechrec.sh
```

Comment:
Modify your script so that it installs to `/usr/local` (rather than `/opt/kaldi`), there are examples of how we build and package things in [docker/usr/local/package](https://github.com/NUbots/NUbots/tree/main/docker/usr/local/package). 

If you install it into `/usr/local` then it will be installed on the robot when you run `./b install` (along with all of the usual optimisations we apply) and you wont need to run the script separately on the robot.

We also already build OpenBLAS for the robots [here](https://github.com/NUbots/NUbots/blob/164974f75c85a3176bcda40dce01d93ccb03bd05/docker/Dockerfile#L178-L181), you should prefer to use this over the one provided by `kaldi` if possible